docs/install-new.md

```diff
@@ -0,0 +1,93 @@
+# Table of Contents
+- [Deploying the infrastructure on Azure](#deploying-the-infrastructure-on-azure)
+- [Setting up Application Gateway Ingress Controller on AKS](#setting-up-application-gateway-ingress-controller-on-aks)
+
+## Deploying the infrastructure on Azure
+
+To create the pre-requisite Azure resources, You can use the following template. It creates:
+1) Azure Virtual Network with 2 subnets.
+2) Azure Application Gateway v2.
+3) Azure Kubernetes Service cluster with required permission to deploy nodes in the Virtual Network.
+4) User Assigned Identity to initialize the aad-pod-identity service and ingress controller.
+5) Set required RBACs.
+
+
+Steps:
+
+1) Create a service principal that will be assigned to aks cluster in the template.
+    ```bash
+    az ad sp create-for-rbac --skip-assignment
+    az ad sp show --id <appId> --query "objectId"
+    ```
+    **Note the appId, password and objectId.**
+
+2) After the above, click to create a custom template deployment. Provide the appId for servicePrincipalClientId, password, objectId in the parameters.
+
+    <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
```

Comment:
@akshaysngupta the URIs point to resources sitting in your fork. Could you create a separate PR, to commit the resources for the templates to this repository and we can than update this PR, to point to content in this repository.